mindmeld/components/dialogue.py

```diff
@@ -1056,6 +1064,16 @@ async def invoke_async(self, request, responder):
         await self.invoke(request, responder)
 
 
+class DialogueResponderSchema(Schema):
+    frame = fields.Dict()
```

Comment:
Can we add a similar form attribute to the DialogueResponderSchema to map to changes in #262

Reply:
done